andela/zinnia-ah-backend
# 164797164 Users can like a comment

#### What does this PR do?
sets user like feature

#### Description of Task to be completed?
- [x] Add association for `commentsLike` table
- [x] setup  association for `comments` table
- [x] setup endpoint to like and unlike a comment

#### How should this be manually tested?
* Clone the repo,
* Run ```npm install``` to install all dependencies,
* Create .env file using the .env.example as a guide and enter you environment variables
* Run `sequelize db:create`  
* Run `sequelize db:migrate` 
* Run `sequelize db:seed:all`   
* Run npm run start:dev,
* send a POST request to `localhost:3000/api/v1/comments/:commentId` using `08fd662d-ed92-419e-8af9-41afd3fb3d87` as the request parameter on Postman
* You should get the screenshot below

#### Any background context you want to provide?
* You need a token to have access to the route

#### What are the relevant pivotal tracker stories? (if applicable)
[#164797164](https://www.pivotaltracker.com/story/show/164797164)

#### Screenshots
<img width="1028" alt="Screenshot 2019-04-14 at 15 00 59" src="https://user-images.githubusercontent.com/15995472/56118169-9ce17b00-5f61-11e9-84b9-1cc3b1e9ab46.png">
<img width="1039" alt="Screenshot 2019-04-14 at 15 01 20" src="https://user-images.githubusercontent.com/15995472/56118173-9f43d500-5f61-11e9-99ac-e40f3d14c5c0.png">
<img width="1043" alt="Screenshot 2019-04-14 at 15 01 39" src="https://user-images.githubusercontent.com/15995472/56118187-a23ec580-5f61-11e9-99fa-506dd7d874e2.png">
<img width="414" alt="Screenshot 2019-04-15 at 10 02 26" src="https://user-images.githubusercontent.com/15995472/56120224-a371f180-5f65-11e9-9a5e-842f4c4bf6c8.png">
